deploy/kubernetes/ingress/k3d-api-ingress.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: api-ingress
+spec:
+  tls:
+  - hosts:
+    - cht-dev.my.local-ip.co
+    secretName: api-tls-secret
+  rules:
+  - host: cht-dev.my.local-ip.co
```

I assume k3d is handling DNS such that these resolve without an IP in the URL (eg `192-168-1-20.local-ip.medicmobile.org`).  Otherwise, updating to use `local-ip.medicmobile.org` per the rest of my feedback:

```suggestion
    - cht-dev.local-ip.medicmobile.org
    secretName: api-tls-secret
  rules:
  - host: cht-dev.local-ip.medicmobile.org
```